src/Chan.cpp

```diff
@@ -423,6 +430,29 @@ void CChan::ModeChange(const CString& sModes, const CNick* pOpNick) {
     }
 }
 
+void CChan::ModeChange(const CString& sModes, const CNick* pOpNick) {
+    VCString vsModes;
+    CString sModeArg = sModes.Token(0);
+    bool colon = sModeArg.TrimPrefix(":");
+
+    // Only handle parameters if sModes doesn't start with a colon
+    if (!colon) {
+        CString sArgs = sModes.Token(1, true);
+
+        while (!sArgs.empty()) {
+            if (sArgs.TrimPrefix(":")) {
```

Comment:
Do you expect colon inside the last parameter as well?

Reply:
This is handling the `+oo Nick1 :Nick2` format whereas the first `TrimPrefix` handles `:+i`. If the first `TrimPrefix` does find a colon at the start, this section is skipped and the entire portion after the colon is passed as the mode character string.